src/pages/BolnisiLanding/index.tsx

```diff
@@ -0,0 +1,565 @@
+          }}
+        >
+          <Box
+            display={"flex"}
+            justifyContent={"space-between"}
+            alignItems={"center"}
+            sx={{ px: 2, [theme.breakpoints.down("sm")]: { flexDirection: "column" } }}
+          >
+            <Box color={theme.isDark ? "#64BCFD" : theme.palette.primary.main} fontSize={"2.25rem"} fontWeight={"bold"}>
+              Recent Transactions
+            </Box>
+            <Box
+              display={"flex"}
+              alignItems={"center"}
+              sx={{ cursor: "pointer", [theme.breakpoints.down("md")]: { mt: theme.spacing(1) } }}
+              height={"44px"}
+              width={"140px"}
+              border={`2px solid ${theme.isDark ? "#64BCFD" : theme.palette.primary.main}`}
+              justifyContent={"center"}
+              borderRadius={"100px"}
+              color={theme.isDark ? "#64BCFD" : theme.palette.primary.main}
+            >
+              <Box fontSize={"1rem"} fontWeight={"500"}>
+                Conformity
+              </Box>
+              <BolnisiDropdown fill={theme.palette.primary.main} />
+            </Box>
+          </Box>
+          <StyledTable
+            columns={columns}
+            data={data}
+            minHeight={780}
+            pagination={{
+              size: 10,
+              total: data.length || 0
+            }}
+            total={{ count: data.length || 0, title: "" }}
+            loading={false}
+          />
+        </Box>
+      </MotionDiv>
+      <MotionDiv initial={{ y: 100, opacity: 0 }} animate={{ y: 0, opacity: 1 }}>
+        <Container maxWidth="md" sx={{ my: "5rem" }}>
+          <Box
+            color={theme.isDark ? "#64BCFD" : theme.palette.primary.main}
+            fontSize={"2.5rem"}
+            lineHeight={"2.625rem"}
+            fontWeight={"bold"}
+          >
+            Discover the full advantages of a future-proof blockchain to empower businessess
+          </Box>
+
+          <Box
+            component={Button}
+            variant="contained"
+            bgcolor={theme.isDark ? "#65BDFE" : theme.palette.primary.main}
+            fontSize={"1rem"}
+            height={"67px"}
+            width={"186px"}
+            borderRadius={"100px"}
+            mt={"2rem"}
+            color={theme.isDark ? "#24262E" : "#fff"}
+            textTransform={"capitalize"}
+          >
+            Read Case Study
+          </Box>
+        </Container>
+      </MotionDiv>
+    </Box>
+  );
+};
+
+const Card = ({ count, title }: { count: number; title: string }) => {
+  const theme = useTheme();
+  return (
+    <Box px={3} py={4}>
+      <Box
+        color={theme.isDark ? "#64BCFD" : theme.palette.primary.main}
+        fontSize={"2.5rem"}
+        lineHeight={"2.875rem"}
+        fontWeight={500}
+        sx={{ textAlign: "left", [theme.breakpoints.down("md")]: { textAlign: "center" } }}
+      >
+        <CountUp end={count} duration={2} />
+        {count === 30 ? "+" : ""}
```

Comment:
Rework this. Adding a plus just when the value is 30? Not a good coding style. If we change the number in the future, this will drive us crazy to find out why the trailing `+` is gone.